### DISCUSSION TITLE
Add `SPI TS CS`, `SPI TS RST`, `SPI TS IRQ` to Universal Touch driver

### DISCUSSION THREAD

**Discussion by s-hadinger** [arendst/Tasmota#21021] [Ideas]
@gemu2015 I love the Universal Touch screen driver. However I would like to offer the possibility of configuring SPI CS/Reset/Irq from the Tasmota template as an alternative to `display.ini`.

Are you ok if I add the 3 GPIO types: `SPI TS CS`, `SPI TS RST` and `SPI TS IRQ`, so the driver would take these values if `display.ini` has value `-1`.

So this would be:
```
:UTI,XPT2046,S1,-1,-1,-1
```

**Comment by gemu2015** [comment]
of course that would be OK !

**Comment by Jason2866** [comment]
@s-hadinger please further discussions for UTouch in #21039
